### ISSUE TITLE
Entering `Mod+[` in Mac+Chrome triggers browser shortcut before key handling in prosemirror

### ISSUE THREAD

**Issue by MH4GF** [ProseMirror/prosemirror#1303]
## Issue details
On Mac+Chrome, keyboard shortcut `Mod+[` triggers browser shortcut before key handling in prosemirror, therefore, even if keymap is defined, it will open the previous page.
This problem occurs only on Mac+Chrome, not on Windows+Chrome/Mac+Safari, etc. Other browsers will browser back if there is no valid command in the keymap defined by prosemirror.

## Steps to reproduce
It is also happening in https://prosemirror.net/ 
Input a nested list like the one below, and entering `Mod+[`
```
- aa
  - bb<cursor>
```

Affected platforms
So far I found that only Mac+Chrome has this issue (tested on windows+chrome, mac+chrome/safari/firefox)

- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer
- [ ] Other

**Comment by marijnh** [comment]
It looks like those keys only take effect when the ProseMirror bindings don't apply (i.e. when `liftListItem` or `sinkListItem` does nothing).

**Comment by MH4GF** [comment]
> t looks like those keys only take effect when the ProseMirror bindings don't apply 

yeah, I understand that is the expected behavior, but Mac + Chrome seems to take effect browsers shortcut before ProseMirror bindings, even if applied.

---

I forgot to mention the version.
Chrome: Version 103.0.5060.134 (Official Build) (x86_64)
Mac: macOS Monterey Version 12.4

**Comment by marijnh** [comment]
What I'm saying is when I try this, that's not what happens.

**Comment by marijnh** [comment]
This issue has been moved to [issue #1303](https://code.haverbeke.berlin/prosemirror/prosemirror/issues/1303) on the [repository's new home](https://code.haverbeke.berlin/prosemirror/prosemirror/).
